ruby/google-protobuf.gemspec

```diff
@@ -14,14 +14,14 @@ Gem::Specification.new do |s|
   }
   s.require_paths = ["lib"]
   s.files       = Dir.glob('lib/**/*.rb')
-  if RUBY_PLATFORM == "java"
-    s.platform  = "java"
+  #if RUBY_PLATFORM == "java"
```

Comment:
If we're not planning on re-instating this commented out block I'd suggest removing it. Otherwise I'd suggest we add a comment to explain why it's commented out.